ooni/nettest.py

```diff
@@ -591,11 +594,16 @@ def generateMeasurements(self):
                 test_instance._setUp()
                 test_instance.summary = self.summary
                 for method in test_methods:
+                    try:
+                        measurement = self.makeMeasurement(
```

Comment:
I placed exception handling here instead of `makeMeasurement` (or it's replacement) to keep makeMeasurement code clean from logging. It's possible to skip bad measurement returning `None`, do you suggest it?

IMHO, the best way is to make the scheduler code exception-neutral instead of fixing exceptions one by one, I've attempted that and that was leading to quite significant refactoring that was probably to invasive for RC stage.